_install-and-configure/upgrade-opensearch/rolling-upgrade.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: default
+title: Rolling Upgrade
+parent: Upgrading OpenSearch
+nav_order: 10
+---
+
+# Rolling Upgrade
+
+Rolling upgrades, sometimes referred to as "node replacement upgrades," can be performed on running clusters with virtually no downtime. Nodes are individually stopped and upgraded in place. Alternatively, nodes can be stopped and replaced, one at a time, by hosts running the new version. During this process you can continue to index and query data in your cluster.
+
+The sample outputs and API responses included in this document were generated in a development environment using Docker containers. Validation was performed by upgrading an Elasticsearch 7.10.2 cluster to OpenSearch 1.3.7; however, this process can be applied to any **N→N+1** version upgrade of OpenSearch on any platform. Certain commands, such as listing running containers in Docker, are included as an aid to the reader but the specific commands used on your host(s) will be different depending on your distribution and host operating system.
+
+This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. For help with [Docker](https://www.docker.com/) or [Docker Compose](https://github.com/docker/compose), refer to the official documentation on their websites.
```

Comment:
Do we mean "refer to the official documentation on the Docker website"? Also, should we refer the reader to https://docs.docker.com/compose/ instead of linking them to GitHub?

Reply:
Good catch but no this was deliberate. Docker Compose is included in Docker Desktop, but not with other distributions of Docker.

Reply:
```suggestion
This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. For help with [Docker](https://www.docker.com/) or [Docker Compose](https://github.com/docker/compose), refer to their official documentation.
```